server/routes.go

```diff
@@ -227,6 +227,11 @@ func GenerateHandler(c *gin.Context) {
 			CreatedAt: time.Now().UTC(),
 			Model:     req.Model,
 			Done:      true,
+			Metrics: api.Metrics{
```

Comment:
Similar to above, let's add this in later if we need to but for this change good structured logging would be better